Models/testmodel1/config.hpp

```diff
@@ -0,0 +1,8 @@
+#include "global_config.hpp"
+
+extern Motor_t motors[];
+extern const uint8_t NUM_MOTORS;
```

Comment:
Why are we using `extern` as opposed to just declaration here and definition in the cpp file?

Reply:
I think this was needed to make the globals accessable from other files, but with constexpr its no longer neccessary.